docs/docs/environment_config.md

```diff
@@ -11,6 +11,7 @@ Environment config stores information related to the environment, such as a part
 An environment config file is a yaml object of key-value pairs configuring Sceptre. The available keys are listed below.
 
 - [iam_role](#iam_role) *(optional)*
+- [iam_profile](#iam_profile) *optional*
```

Comment:
Could we rename `iam_profile` to just `profile` throughout this PR? Profiles don't exist within IAM - they are just using in managing credentials on your local machine.